src/toil_lib/urls.py

```diff
@@ -8,11 +9,15 @@
 from toil_lib.programs import docker_call
 
 
-def download_url(url, work_dir='.', name=None, s3_key_path=None, cghub_key_path=None):
+_log = logging.getLogger(__name__)
```

Comment:
The other modules use _log, I'm not sure why either. This module was just using ```print``` so I added a logger and followed the convention